common/baselines/librispeech/ls100/gmm/baseline_args.py

```diff
@@ -0,0 +1,428 @@
+                    channels=20, warping_function="mel", f_max=8000
+                ),
+                "normalize": True,
+                "normalization_options": None,
+                "without_samples": False,
+                "samples_options": samples_options,
+                "cepstrum_options": {
+                    "normalize": False,
+                    "outputs": 16,  # this is the actual output feature dimension
+                    "add_epsilon": not dc_detection,  # when there is no dc-detection we can have log(0) otherwise
+                    "epsilon": 1e-10,
+                },
+                "fft_options": None,
+            },
+        },
+        "energy": {
+            "energy_options": {
+                "without_samples": False,
+                "samples_options": samples_options,
+                "fft_options": {},
+            }
+        },
+    }
+
+    return RasrInitArgs(
+        costa_args=costa_args,
+        am_args=am_args,
+        feature_extraction_args=feature_extraction_args,
+        default_mixture_scorer_args={"scale": 0.3},
+    )
+
+
+def get_monophone_args():
+    linear_alignment_args = {
+        "minimum_segment_length": 0,
+        "maximum_segment_length": 6000,
+        "iterations": 5,
+        "penalty": 0,
+        "minimum_speech_proportion": 0.7,
+        "save_alignment": False,
+        "keep_accumulators": False,
+        "extra_merge_args": None,
+        "extra_config": None,
+        "extra_post_config": None,
+    }
+
+    monophone_training_args = {
+        "name": "mono",
+        "feature_flow": "mfcc+deriv+norm",
+        "feature_energy_flow_key": "energy,mfcc+deriv+norm",
+        "align_iter": 75,
+        "splits": 10,
+        "accs_per_split": 2,
+        "dump_alignment_score_report": True,
+    }
+
+    monophone_recognition_args = {
+        # GmmSystem.recognition() args:
+        "iters": [10],
+        "lm_scales": [10],
+        "optimize_am_lm_scale": True,
+        # meta.System.recog() args:
+        "feature_flow": "mfcc+deriv+norm",
+        "pronunciation_scales": [1.0],
+        "lm_lookahead": True,
+        "lookahead_options": None,
+        "create_lattice": True,
+        "eval_single_best": True,
+        "eval_best_in_lattice": True,
+        "search_parameters": {
+            "beam-pruning": 15.0,
+            "beam-pruning-limit": 100000,
+            "word-end-pruning": 0.5,
+            "word-end-pruning-limit": 15000,
+        },
+        "parallelize_conversion": False,
+        "lattice_to_ctm_kwargs": {},
+        "rtf": 20,
+        "mem": 4,
+        "use_gpu": False,
+    }
+
+    return gmm_system.GmmMonophoneArgs(
+        linear_alignment_args, monophone_training_args, monophone_recognition_args
+    )
+
+
+def get_cart_args(
+    use_stress_marker: bool = False,
+    max_leaves: int = 12001,
+    min_obs: int = 1000,
+    hmm_states: int = 3,
+    feature_flow: str = "mfcc+deriv+norm",
+    add_unknown: bool = True,
```

Comment:
should the default here maybe match the default in `common/baselines/librispeech/data.py` where in the lexicon unknown phoneme is added iff `not use_g2p_training` with `use_g2p_training=True` by default --> no unknown phoneme by default.
```suggestion
    add_unknown: bool = False,
```